docs: minor update (#297)
pymmcore-plus/pymmcore-plus: docs/trableshooting.md

> # Trableshooting

Reviewer: ```suggestion
# Troubleshooting
```

## Micro-Manager directory not found

```sh
pymmcore-plus - ERROR - (_util.py:131) could not find micromanager directory. Please run 'mmcore install'
```

If you tried to create a [`pymmcore_plus.CMMCorePlus`](api/cmmcoreplus.md) instance and got an

> error similar the one above, it probably means that you don't have Micro-Manager installed on your computer

Reviewer: ```suggestion
error similar the one above, it means that pymmcore-plus was unable to find micro-manager on on your system.
```

> (for example, you can run `mmcore install` to install the latest version of Micro-Manager).

Reviewer: ```suggestion
Please run `mmcore install` to install the latest version of Micro-Manager.
```
